iOS/_Stream Temp/ChatClient+Extensions.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ChatClientManager.swift
+//  ChatClientManager
+//
+//  Created by Martin Young on 9/9/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+extension ChatClient {
```

Comment:
No need for the chat manager any more. Just an extension with some convenience functions.